e2e/cypress/integration/messaging/visual_verification_of_tooltips_on_top_nav,_channel_icons,_posts_spec.js

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+describe('Messaging', () => {
+    let testTeam;
+
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+
+            // # Set up Demo plugin
+            cy.apiInstallPluginFromUrl('https://github.com/mattermost/mattermost-plugin-demo/releases/download/v0.8.0/com.mattermost.demo-plugin-0.8.0.tar.gz', true);
+            cy.apiEnablePluginById('com.mattermost.demo-plugin');
+
+            // # Login as regular user
+            cy.apiLogin(user);
+
+            // # Set up test channel with a long name
+            cy.apiCreateChannel(testTeam.id, 'channel-test', 'Public channel with a long name');
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+        });
+    });
+    after(() => {
```

Comment:
Please separate with empty line for consistency.
```suggestion
    });

    after(() => {
```